### PR TITLE
 [TBDGEN] remove stale tapi files

### PR DESCRIPTION
**Pull request by cyndyishida**
This directory was removed on master in https://github.com/apple/swift/pull/27073, remove the persisted files from master-next, this doesn't get built anywhere anyway.

### CODE REVIEW

**Review by harlanhaskins — APPROVED** [review]

**Comment by cyndyishida** [comment]
planning on waiting until CI works to test, then merge this in.

**Review by ributzka — APPROVED** [review]

**Comment by compnerd** [comment]
@swift-ci please smoke test and merge

**Pull request closed without merging**
